hathor/transaction/storage/transaction_storage.py

```diff
@@ -127,6 +127,7 @@ def __init__(self):
 
```

Comment:
```python
        self._dir_dep_index: Dict[bytes, _DirDepValue] = {}
```

> I feel we don't need this direct mapping. But we can discuss it further in the sync-v2 PR.

I'm bringing this up from https://github.com/HathorNetwork/hathor-core/pull/226#discussion_r654180481 (which was moved to #238).

Reply:
Fixed in #292 .